salesagility/SuiteCRM
# Fixed #4155 - Project Template and Task creation gets a blank screen and relationship errors in the log

## Description



When creating a project from a project template the user is able to skip entering a start date causing errors and a white page. 

## Motivation and Context

This change ensures that a user has entered a start date when creating a project from project templates by making it a mandatory field.

Issue reference: #4155

## How To Test This

1. Create a Project Template with some Tasks.
2. Open action-menu and click "Create Project".
3. In the pop-up window give the new project a name and select the check box to include the tasks.
4. Check that without entering a date the task menu will not allow you to create a project.
5. Check that after clicking "Create Project" a project is correctly created.

## Types of changes


### Final checklist
